### PR TITLE
Rnaseq data demo

### PR DESCRIPTION
**Pull request by jaybee84**
This is the Rmd file for the HTML (syn18693228)

### CODE REVIEW

File: data_demos/rnaseq-data-introduction.Rmd [hunk 1]
```diff
@@ -0,0 +1,322 @@
...
+fileEntity <- synGet('syn18137070')
+# Download the data file
+genes.with.meta<-read.csv(gzfile(fileEntity$path))
+```
+
+# Explore the data files
+
+## Structure of the file
+
+* The Structure of the file refers the column names and the type of variables stored in it:
```
Comment (sgosline) [inline data_demos/rnaseq-data-introduction.Rmd]: This might be prettier as a table - can you do that in Rmd?

**Comment by sgosline** [comment]
This is great! As far as the shared functions go, I think it might be easier to put these functions into an R folder so that we can just load them as a package. This will eliminate the need to `source` the right file. It will also give us the ability to easily install this package into the docker image later on.

File: data_demos/rnaseq-data-introduction.Rmd (hunk 1 above)
Comment (jaybee84) [reply data_demos/rnaseq-data-introduction.Rmd]: I can try to make a knitr table and see how it looks...

**Comment by allaway** [comment]
For easy tables I am a big fan of the `pander` library.

**Comment by jaybee84** [comment]
Regarding the functions: I have actually put the adapted functions right into the final Rmd file... The "handy functions" file is something I made earlier but have not sourced it in the current Rmd to avoid source issues.

**Comment by allaway** [comment]
@sgosline I see where you are coming from, though I think as an interactive notebook walkthrough it is valuable for people to see the functions in front of them as they run the script, it's also easier for them to copy and paste and modify them if they need to do that. 

I also have been viewing these markdowns as more of a data exploration tutorial for people to explore the data pre-hackathon, rather than code they'd actively use during (with the exception, perhaps, of the synGet's for specific datasets)?

**Comment by jaybee84** [comment]
I refrained from echoing functions/code in the final HTML (except synGet) for easier readability and also to encourage participants to explore the data using the tools of their choosing. Happy to change it to display the code.. 

In that case, should we start preparing parallel notebooks in python for people who would choose to do the challenge in python?

**Comment by allaway** [comment]
I don't know whether we need to include the code in the final HTML, I agree
that hiding it makes it more readable.

My thought was that users would run through the actual .Rmd or .ipynb if
they want to walk through the code themselves. The HTML can be provided for
those that just want to look but not actually run the code.

Agree we should have parallel ipynbs. I don't know whether we'll be able to
do all the same analyses (e.g. GSVA) across both languages, but at least
showing people how to get the data in both languages will be important.

**Comment by xdoan** [comment]
You can fold the code in markdown documents, so people can expand it if they want to see it. `code_folding: hide` under html_document is all you need

**Comment by xdoan** [comment]
I like pander for simple tables, but if it has many rows/columns (can't see the html doc) DT::datatables offers interactive tables that people can sort/filter etc on https://rstudio.github.io/DT/

**Comment by jaybee84** [comment]
Like the idea of code folding in the HTML. May make it easier for the participants to have a unified doc to go to for both code and viz.

**Comment by xdoan** [comment]
here's an example html https://www.synapse.org/#!Synapse:syn18361348 , you can show/hide all code or show/hide code for a specific chunk (and not have to do echo = F for each new chunk)

**Review by allaway — APPROVED** [review]
looks great to me! Feel free to merge. I'll add you as a collaborator on this project so you can also review and merge.

**Pull request merged**